### PR TITLE
[IOAPI] Save .h and .mod files

### PR DESCRIPTION
**Pull request by ctessum**

### CODE REVIEW

**Comment by ctessum** [comment]
I got the windows build to work, but it says it is not able to find -lnetcdff when converting it to a dynamic library. Is there an obvious fix for this problem that I'm missing?

File: I/IOAPI/build_tarballs.jl [hunk 1]
```diff
@@ -24,21 +24,30 @@ mkdir $BINDIR
...
 
-make fixed_src
 make # Parallel make (-j > 1) does not work
 
+cp *.h ${includedir} # C header files
 cp fixed_src/* ${includedir} # FORTRAN .EXT (include) files
 
-gfortran -shared -fPIC -fopenmp -o ${libdir}/libioapi.${dlext} -Wl,$(flagon --whole-archive) ${BINDIR}/libioapi.a -Wl,$(flagon --no-whole-archive | cut -d' ' -f1) -lnetcdf -lnetcdff
+# Convert static library to dynamic library
+gfortran -shared -fPIC -fopenmp -o ${libdir}/libioapi.${dlext} -Wl,$(flagon --whole-archive) ${BINDIR}/libioapi.a -Wl,$(flagon --no-whole-archive | cut -d' ' -f1) -lnetcdf -Wl,$(flagon --no-whole-archive | cut -d' ' -f1) -lnetcdff
```
Comment (giordano) [inline I/IOAPI/build_tarballs.jl:50]:
```suggestion
gfortran -shared -fPIC -fopenmp -o ${libdir}/libioapi.${dlext} -L${libdir} -Wl,$(flagon --whole-archive) ${BINDIR}/libioapi.a -Wl,$(flagon --no-whole-archive | cut -d' ' -f1) -lnetcdf -Wl,$(flagon --no-whole-archive | cut -d' ' -f1) -lnetcdff
```

Comment (giordano) [reply I/IOAPI/build_tarballs.jl:50]: Even with this, I get other errors:
```
sandbox:${WORKSPACE}/srcdir/ioapi-3.2/ioapi # gfortran -shared -fPIC -fopenmp -o ${libdir}/libioapi.${dlext} -L${libdir} -Wl,$(flagon --whole-archive) ${BINDIR}/libioapi.a -Wl,$(flagon --no-whole-archive | cut -d' ' -f1) -lnetcdf -Wl,$(flagon --no-whole-archive | cut -d' ' -f1) -lnetcdff
/workspace/srcdir/ioapi-3.2/tmp/libioapi.a(iobin3.o):iobin3.c:(.text+0x16b2): undefined reference to `fsync'
/workspace/srcdir/ioapi-3.2/tmp/libioapi.a(iobin3.o):iobin3.c:(.text+0x4040): undefined reference to `fsync'
/opt/x86_64-w64-mingw32/bin/../lib/gcc/x86_64-w64-mingw32/4.8.5/../../../../x86_64-w64-mingw32/bin/ld: /workspace/srcdir/ioapi-3.2/tmp/libioapi.a(iobin3.o): bad reloc address 0x870 in section `.rdata'
collect2: error: ld returned 1 exit status
```
`fsync` is a Unix function, I'm not sure this library is supposed to work on Windows at all: https://www.cmascenter.org/ioapi/

> System Requirements
> O/S | Linux or UNIX or MacOS

Comment (ctessum) [reply I/IOAPI/build_tarballs.jl]: Oh well, it was worth a try. I'd still like to add in the other changes so I will remove the windows build for now.

**Pull request merged**